### DISCUSSION TITLE
Providing Certificate to users who watched recording of Webinar

### DISCUSSION THREAD

**Discussion by mahajanmahesh935** [Sunbird-Ed/Community#685] [Feature request]
Hello Team,
We are creating "Webinars" for users in which user will enrol and only enrolled users can join the webinar and we will provide the certificate to users who attended webinar online based on the attendance of the webinar but in some case if enrolled user not able join the webinar, then we will provide the recording of the webinar on portal so that user can watch recording of webinar and we want to provide the certificate to users who watched recording.

so we need help on following questions
1. **How we can track the user who watched the recording**?
2. **can we track the duration of watching recording?** if yes then how we can track it?
3. **How we can mark this content completed if recording watched by user**

We appreciate any insights. Thank you!

@vinukumar-vs ,@rajeevsathish, @AmiableAnil , @Krishnaj20, @chitranshu-keshav, @SanthoshVasabhaktula, @gandham-santhosh , @coolbung @komalm @manishakapadnis

**Comment by mahajanmahesh935** [comment]
Hi team,
Your insights and assistance would be highly appreciated in resolving these matters. Looking forward to your response.

**Reply by vinukumar-vs** [reply]
@mahajanmahesh935 If you configure "webinar" as primary-category. You can attach the property "trackable" to it. Based on this property, the progress will be attached to the collection.

I would suggest to this this on the Knowlg and Lern BB discussions thread. Because they are the primary owners of this feature.

I would suggest you go through [Sunbird AI - Askdot](https://bot.sunbird.org/) by asking appropriate questions
<img width="1480" alt="image" src="https://github.com/Sunbird-Ed/Community/assets/4496096/94d2568d-0550-41d2-a69d-bdb1f8e9a99f">

@mohitga [@amitpriyadarshi](https://github.com/amitpriyadarshi)

**Comment by anisha147** [comment]
Hi @mahajanmahesh935 

We hope the suggestions shared by @vinukumar-vs helped you resolve the matter. If so, could you please close this thread from your end?

cc: @vinukumar-vs
